config.yml

```diff
@@ -9,12 +9,15 @@
 concept_source:
   status:
     - identical
+    - similar # in iev-data
     - modified
     - restyle
     - context-added
     - generalisation
     - specialisation
     - unspecified
+    - related # in iev-data
+    - not_equal # in iev-data
```

Comment:
Since we have `context-added`, can we use `not-equal` with hyphen?

Reply:
I've updated the `context-added` to `context_added` with an underscore because everything else was with an underscore.